setup.py

```diff
@@ -36,7 +36,7 @@ def run(self):
 
 
 setup(name='gemd',
-      version='0.9.2',
+      version='0.9.3',
```

Comment:
`change_scope` is a new public method, so I think this should be a minor version bump.